slemicro/create_vm.sh

```diff
@@ -159,6 +159,7 @@ if [ -f ${BASEDIR}/combustion/script ]; then
 	
 	# Copy all combustion related files to the final iso destination parsing the vars
 	for file in ${BASEDIR}/combustion/*.sh; do
+		[ -f "$file" ] || break
 		FILENAME=$(basename ${file})
 		envsubst < ${file} > ${TMPDIR}/combustion/${FILENAME}
 		chmod a+x ${TMPDIR}/combustion/${FILENAME}
```

Comment:
I prefer to use something like:

```
	if ls ${BASEDIR}/combustion/*.sh >/dev/null 2>&1; then
		# Copy all combustion related files to the final iso destination parsing the vars
		for file in ${BASEDIR}/combustion/*.sh; do
			FILENAME=$(basename ${file})
			envsubst < ${file} > ${TMPDIR}/combustion/${FILENAME}
			chmod a+x ${TMPDIR}/combustion/${FILENAME}
			echo "./${FILENAME}" >> ${TMPDIR}/combustion/script
		done
	fi
```

It avoids the loop if there are no files and personally I don't like breaks :D

Reply:
I like it better, change made.